src/App.js

```diff
@@ -1,25 +1,91 @@
-import logo from './logo.svg';
 import './App.css';
+import contacts from "./contacts.json"
+import { useState } from 'react';
+
 
 function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+  
+
+  const [contactInfo, setContacts] = useState(contacts.slice(0,5))
+
+  const randomArr = [...contacts].filter(contact => !contactInfo.includes(contact))
+
+  
+  console.log(randomArr);
+
+
+  const addRandomContact = () => {  
+
+    const intRandom = Math.floor (Math.random() * randomArr.length);
+    const addContact = randomArr[intRandom];
```

Comment:
Also I would add a `if` statement to guard the function when you already added everyone, otherwise the application will break